app/controllers/concerns/accounts/current_user.rb

```diff
@@ -103,13 +103,19 @@ def logout_user
   end
 
   # Redirect the user according to the logout scheme
-  def perform_post_logout(_prev_session)
+  def perform_post_logout(prev_session, prev_user)
     if Setting.login_required? && omniauth_direct_login?
```

Comment:
Originally this block is only there to prevent an instant re-login after logging out with direct login enabled.
Even with it enabled the user should be redirected to the configured slo_target_url, though, shouldn't they?

If the SSO provider then redirects back to OpenProject we have to make sure that OpenProject doesn't try to initiate another login in this case.

Reply:
I think even in the case that OpenProject decides to redirect to the idP again, if you were globally logged out, you would only be redirected to the login form so I believe this would be o kay.

In any case, an omniauth direct login should still be allowing the SLO callback. Will fix that